opm/material/densead/Evaluation.hpp

```diff
@@ -47,26 +48,26 @@ namespace DenseAd {
  * \brief Represents a function evaluation and its derivatives w.r.t. a fixed set of
  *        variables.
  */
-template <class ValueT, unsigned numVars>
```

Comment:
Well, yes, but then I prefer int, so I am probably willing to fix any warning fallout from this...